src/config/experiment.js

```diff
@@ -16,11 +16,11 @@ exptBlock1.get_reward = [true, true, true, false];
 exptBlock1.num_breaks = 3;
 
 // debug settings where there are only 2 trials in main task block
-// exptBlock1.probs = ["100%"];
-// exptBlock1.counterbalance = true;
-// exptBlock1.value = [3];
-// exptBlock1.effort = [5];
-// exptBlock1.keys = ["q", "p", "m"];
-// exptBlock1.get_reward = [true];
+/*exptBlock1.probs = ["100%"];
```

Comment:
this looks like the debug settings -- should be commented out again

Reply:
They are commented out, just not in the way they were originally, how should I change these in the pull request?

Reply:
you can push again to update the PR. maybe they're commented out as of now, but the state your code was in for the PR is saying that they weren't commented out at all as the // were removed for those lines..

Reply:
@moblim ooh ok nvm, i see what you're saying now! sorry about that. in the review GUI they have on here it was hard to see the * and it simply had looked like the // were removed completely and that none of it was being commented out. in any event, somewhat trivial but still good to match repo so that it's not registered as an actual change